forge-gui/res/cardsfolder/upcoming/caught_in_a_parallel_universe.txt

```diff
@@ -0,0 +1,9 @@
+Name:Caught in a Parallel Universe
+ManaCost:no cost
+Types:Phenomenon
+T:Mode$ PlaneswalkedTo | ValidCard$ Card.Self | Execute$ TrigRepeat | TriggerDescription$ When you encounter CARDNAME, each player chooses a creature controlled by the player to their left. Each player creates a token that's a copy of the creature they chose, except it has menace. (Then planeswalk away from this phenomenon.)
+SVar:TrigRepeat:DB$ RepeatEach | RepeatPlayers$ Player | StartingWithActivator$ True | ChangeZoneTable$ True | RepeatSubAbility$ DBChooseCreature | SubAbility$ PWAway
+SVar:DBChooseCreature:DB$ ChooseCard | Defined$ Remembered | Choices$ Creature.ControlledByPlayerInTheDirection_Left_Player.IsRemembered | ChoiceTitle$ Choose a creature controlled by the player to your left | Mandatory$ True | RememberChosen$ True | SubAbility$ DBToken
```

Comment:
do this without extra Remember or it will include previous choices

Reply:
or maybe manual cleanup would be better in case a player doesn't control any creature